README.md

```diff
@@ -1,94 +1,142 @@
-
-* build
-* schema
-* version
-* install
-* upgrade
-* invoke
-* uninstall
-
-Each of these command implementations have a corresponding Mixin implementation in the `pkg/docker` directory. Each of the commands above is wired into an empty implementation in `pkg/docker` that needs to be completed. In order to build a new Mixin, you need to complete these implementations with the relevant technology. For example, to build a [Cloud Formation](https://aws.amazon.com/cloudformation/) mixin, you might implement the methods in `pkg/docker` using the [AWS Go SDK](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudformation/).
-
-## Provided capabilities
-
-This skeleton mixin project brings some free capabilities:
-
-### File System Access and Context
-
-Porter provides a [Context](https://github.com/deislabs/porter/tree/main/pkg/context) package that has helpful mechanisms for accessing the File System using [spf13/afero](https://github.com/spf13/afero). This makes it easy to provide mock File System implementations during testing. The Context package also provides a mechanism to encapsualte stdin, stdout and stderr so that they can easily be passed from `cmd/docker` code to implementing `pkg/docker` code.  
-
-### Template and Static Asset Handling
-
-The project already includes [Packr V2](https://github.com/gobuffalo/packr/tree/master/v2) for dealing with static files, such as templates or other content that is best modeled outside of a Go file. You can see an example of this in `pkg/docker/schema.go`.
-
-### Basic Schema
-
-The project provides an implementation of the `docker schema` command that is mostly functional. To fully implement this for your mixin, you simply need to provide a valid JSON schema. For reference, consult `pkg/docker/schema/schema.json`.
-
-### Basic Tests
-
-The project provides some very basic test skeletons that you can use as a starting point for building tests for your mixin.
-
-### Makefile
-
-The project also includes a Makefile that will can be used to both build and install the mixin. The Makefile also includes a TODO `publish` target that shows how you might publish the mixin and generate an mixin feed for easily sharing your mixin.
\ No newline at end of file
+# A Docker Mixin for Porter
+
+This is a Docker mixin for Porter. The mixin provides the Docker CLI. 
+
+## Mixin Declaration
+
+To use this mixin in a bundle, declare it like so:
+
+```yaml
+mixins:
+- docker
+```
+
+## Mixin Commands
+The commands available are docker pull, push, build, run, remove, and login. 
+
+## Mixin Syntax
+The same syntax applies for install, upgrade, and uninstall.
+
+### Docker pull
+```yaml
+- docker:
+    description: 
+    pull:
+      name:
+      tag:
+```
+
+### Docker push
+```yaml
+- docker:
+    description: 
+    push:
+      name:
+      tag:
+```
+
+### Docker build
+```yaml
+- docker:
+    description: 
+    build:
+      tag: 
+      file: OPTIONAL
```

Comment:
Since we are going to fill in example values, this should change to be a yaml comment, like you are using elsewhere